pyof/foundation/network_types.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, tlv_type=127, value=BinaryData()):
         """
         super().__init__()
         self.tlv_type = tlv_type
-        self._value = value
+        self._value = BinaryData(value) if isinstance(value, bytes) else value
```

Comment:
Well noticed. We tried to use in the beginning, but didn't work because the sizes are different from LLDP TLV.